lib/main/lang/projectService.ts

```diff
@@ -938,22 +938,26 @@ interface GetOutputJsStatusResponse {
 export function getOutputJsStatus(query: FilePathQuery): Promise<GetOutputJsStatusResponse> {
     consistentPath(query);
     var project = getOrCreateProject(query.filePath);
-    var output = getRawOutput(project, query.filePath);
-    if (output.emitSkipped) {
-        if (output.outputFiles && output.outputFiles.length === 1) {
-            if (output.outputFiles[0].text === building.Not_In_Context) {
+    return getRawOutputPostExternal(project, query.filePath)
```

Comment:
`runExternalTranspiler` (which is called by `getRawOutputPostExternal`) is kind of a misnomer. It's a no-op if an external transpiler isn't configured to run. Ref: https://github.com/laurelnaiad/atom-typescript/blob/iss737/lib/main/lang/modules/building.ts#L168